tests/test_file_listing.py

```
@@ -248,39 +351,199 @@ def mock_is_git_repo():
         yield mock
 
 
+@pytest.fixture
+def mock_os_path(monkeypatch):
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_os_path`

(ANN201)